cypress/e2e/help-section/helpAreas.feature

```diff
@@ -5,8 +5,13 @@ Feature: Help Section - To use the help areas in dBildungscloud
 
     As a user I want to use the help areas in the header and sidebar so that I can find help when needed
 
-    Scenario Outline: User can use the help area in the (header, sidebar), search article and submit an issue via contact form
-        Given I am logged in as a '<user>' at '<namespace>'
+    Scenario Outline: User can use the help area in the (header, sidebar), search article and submit an issue and a request via contact form
```

Comment:
I don't think we have help section (either article search, issue submission or sending request) in header area anymore? 

In that case update description from here and above